aion/web/channels/room_channel.ex

```diff
@@ -21,7 +21,7 @@ defmodule Aion.RoomChannel do
   @user_list_topic "user_list"
 
   @spec join(String.t, %{}, UserSocket.t) :: {:ok, UserSocket.t}
-  def join("rooms:" <> room_id, _params, socket) do
+  def join("room:" <> room_id, _params, socket) do
```

Comment:
I also renamed rooms to room according to [the docs](https://hexdocs.pm/phoenix/channels.html#tying-it-all-together) once again. I know this PR is not about refactoring the whole elixir stuff, but since it was dealing with messages heavily I thought this might be a good time to finally do it.